app/components/GettingStartedModal/localStorage.ts

```diff
@@ -1,7 +1,5 @@
-export const isGettingStartedModalDisabledInLocalStorage = (): boolean => {
-    const modalDisabled = localStorage.getItem('getting-started-modal-disabled');
-    return modalDisabled === undefined || modalDisabled !== 'true';
-};
+export const isGettingStartedModalDisabledInLocalStorage = (): boolean =>
+    localStorage.getItem('getting-started-modal-disabled') === 'true';
 
 export const disableGettingStartedModalInLocalStorage = (): void => {
     localStorage.setItem('getting-started-modal-disabled', 'true');
```

Comment:
I'd suggest keeping function names shorter - given the context in which they are defined (file name) I think `InLocalStorage` suffix is not needed and actually redundant.

```suggestion
export const isGettingStartedModalDisabled = (): boolean =>
    localStorage.getItem('getting-started-modal-disabled') === 'true';

export const disableGettingStartedModal = (): void => {
    localStorage.setItem('getting-started-modal-disabled', 'true');
```

Reply:
fixed